content/coronavirus_landing_page.yml

```diff
@@ -81,14 +81,16 @@ content:
       sub_sections:
         - title:
           list:
-            - label: Full guidance on coronavirus testing and eligibility
+            - label: Guidance on testing for essential workers and care homes 
               url: /guidance/coronavirus-covid-19-getting-tested
-            - label: Apply for a coronavirus test if you’re eligible
+            - label: Apply for a test if you have coronavirus symptoms 
+              url: https://www.nhs.uk/conditions/coronavirus-covid-19/testing-for-coronavirus/ask-for-a-test-to-check-if-you-have-coronavirus/
+            - label: Apply for a test if you are an essential worker
               url: /apply-coronavirus-test
-            - label: Book a coronavirus test if you have a verification code
-              url: https://test-for-coronavirus.service.gov.uk/appointment
             - label: Apply for tests for a care home
               url: /apply-coronavirus-test-care-home
+            - label: Book a test if you have a verification code
+              url: https://test-for-coronavirus.service.gov.uk/appointment
```

Comment:
I know this was here before, but what happened to "Start on GOV.UK"? Shouldn't we be linking to start pages rather than direct to service domains? 🤔

Reply:
Oh I suppose we are already on GOV.UK at the point of seeing this link. Just like a start page. Cool. Thanks for catching me, @helengds!